test/TestConfig/resources/excludes/latest_exclude_21.txt

```diff
@@ -31,6 +31,18 @@ org.openj9.test.nestmates.NestAttributeTest:testGetNestMembersAPINestMemberWrong
 org.openj9.test.nestmates.NestAttributeTest:testGetNestMembersAPINestMemberWrongNestHost NA generic-all
 org.openj9.test.nestmates.NestAttributeTest:testGetNestMembersAPINestMemberWrongNestHost2 NA generic-all
 
+org.openj9.test.jep442.downcall.InvalidDownCallTests:test_heapSegmentForStructArgument NA zos_390-64
```

Comment:
Instead of `NA`, do we have an issue number to reference? javanext/issues/441?

Reply:
Replaced with `javanext/issues/441` as suggested above.